kernel/selinux/Makefile

```diff
@@ -2,6 +2,14 @@ obj-y += selinux.o
 obj-y += sepolicy.o
 obj-y += rules.o
 
+ifeq ($(shell [ $(VERSION) -lt 5 -o \( $(VERSION) -eq 5 -a $(PATCHLEVEL) -lt 10 \) ]; echo $$?),0)
+  ifeq ($(shell grep -q " current_sid(void)" $(srctree)/security/selinux/include/objsec.h; echo $$?),0)
+  ccflags-y += -DKSU_COMPAT_HAS_CURRENT_SID
+  endif
+  ifeq ($(shell grep -q "struct selinux_state " $(srctree)/security/selinux/include/security.h; echo $$?),0)
+  ccflags-y += -DKSU_COMPAT_USE_SELINUX_STATE
```

Comment:
make this to KSU_COMPAT_HAS_SELINUX_STATE, and we can use

#ifdef KSU_COMPAT_HAS_SELINUX_STATE || version > 5.10
#define KSU_COMPAT_USE_SELINUX_STATE
#endif

Reply:
https://github.com/tiann/KernelSU/pull/402/commits/6bf07bf51559512439290da07af1381faa63de14 done.

I put it in `selinux.h`, since both `selinux.c` and `rules.c` have the same check. Let me know if it is inappropriate.